web/handlers/v2/genomics/lineage.py

```diff
@@ -0,0 +1,113 @@
+from web.handlers.genomics.base import BaseHandler
+
+from tornado import gen
+import pandas as pd
+from web.handlers.genomics.util import (
+    create_nested_mutation_query,
+    get_total_hits
+)
+
+import re
+
+
+class LineageMutationsHandler(BaseHandler):
+    gene_mapping = {
+        "orf1a": "ORF1a",
+        "orf1b": "ORF1b",
+        "s": "S",
+        "orf3a": "ORF3a",
+        "e": "E",
+        "m": "M",
+        "orf6": "ORF6",
+        "orf7a": "ORF7a",
+        "orf7b": "ORF7b",
+        "orf8": "ORF8",
+        "n": "N",
+        "orf10": "ORF10"
+    }
+
+    name = 'lineagemutations'
+    kwargs = dict(BaseHandler.kwargs)
+    kwargs['GET'] = {
+        'pangolin_lineage': {'type': str, 'default': ''},
+        'frequency': {'type': float, 'default': None},
+        'gene': {'type': str, 'default': ''}
+    }
+
+    @gen.coroutine
```

Comment:
is this still the right way, or we should better use `async def` now? @remoteeng00

Reply:
I did try with `async def` and it works, so I switch to use `async def`